### ISSUE TITLE
Incorrect error: Camelcase `SimpleITK` imported as lowercase `sitk` Ruff N813

### ISSUE THREAD

**Issue by MattTheCuber** [astral-sh/ruff#12750]
Example code:
```python
import SimpleITK as sitk
```

Enable the Ruff rule `N813` shows an error for this line. It is correct, however this is the standard and recommended import method according to the [official docs](https://simpleitk.org/), similar to numpy and pandas. I found no issues mentioning SimpleITK.

**Comment by charliermarsh** [comment]
Thanks for filing. I'm hesitant to add library-specific exceptions here. I think my advice would be to add `SimpleITK` to [`ignore-names`](https://docs.astral.sh/ruff/settings/#lint_flake8-self_ignore-names):

```toml
[tool.ruff.lint.flake8-self]
ignore-names = ["SimpleITK"]
```

**Comment by charliermarsh** [comment]
Gah sorry, I linked to the wrong setting.

**Comment by charliermarsh** [comment]
Edited my comment to reflect the correct version.
